package-lock.json

```diff
@@ -3880,61 +3880,46 @@
             }
         },
         "node_modules/@cypress/request": {
```

Comment:
Do you need changes in this file?

Reply:
It is required by the cypress update, the pipeline would crash without it